jvm/publish.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'com.github.dcendents.android-maven'
 
-version = '0.0.8'
+version = System.getenv("GITHUB_REF")
```

Comment:
Q: Should it be better to have another env variable "GITHUB_REF" and get the version from that one?
Also, is ```GITHUB_REF``` the release version (in this context)?

Reply:
yeah GITHUB_REF is the release version as specified by @vvmnnnkv